app/models/provider.rb

```diff
@@ -86,8 +86,14 @@ class Provider < ApplicationRecord
            primary_key: :provider_code,
            inverse_of: :accrediting_provider
 
+  # the accredited_providers that this provider is a training_provider for
   has_many :accrediting_providers, -> { distinct }, through: :courses
```

Comment:
Is the right name for this `qualification_providers`?

Reply:
🤷‍♂ Who knows? That's existing code. We're pretty settled on accredited_provider and training_provider as the distinction but as we've discussed, plenty of flavours. Paul G suggested training provider which had a certain ring to it so we went with that. I've got a tech debt 'note' to make it consistent. Open to suggestions on where to stick that 😆

Reply:
I'm starting to keep my tech debt on my own Trello board, but we might start up a backlog Trello board that the whole team can use. Discussion to come.

The reason I asked here, though, is that I'm thinking what we'll be calling it in the "V3" API, the one that'll be more stable and for public consumption. Thinking about this a little more, I'm not sure we need `qualification_providers`, since we have:

- **accredited_body** - the provider that is a accredited by the DfE to provide qualifications for a course
- **training_provider** - the provider chosen by the accredited_body to provider the training for the course

As for the relationship above, sounds like it should be `accredited_bodies`.